### PR TITLE
Feat/rapid 3392 overhaul examples

### PR DESCRIPTION
**Pull request by LinoGiger**

### CODE REVIEW

File: docs/custom_orders.md [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
 
 ```
-pip install rapidata -U
+pip install -U rapidata
 ```
 
 ## Usage
 
-Orders are managed through the [`RapidataClient`](reference/rapidata/rapidata_client/rapidata_client.md#rapidata.rapidata_client.rapidata_client.RapidataClient).
+Orders are managed through the `RapidataClient`.
```
Comment (kannwism) [inline docs/custom_orders.md:22]: Why did you delete the link?

File: docs/custom_orders.md [hunk 2]
```diff
@@ -1,44 +1,45 @@
 # Custom order examples
+Directly ask real humans to classify your data. This guide will show you how to create a classification order using the Rapidata API.
```
Comment (kannwism) [inline docs/custom_orders.md:2]: This file is now for the "advanced" user, right? It should be clear that we are asking real humans at this point no?

File: docs/custom_orders.md [hunk 3]
```diff
@@ -1,44 +1,45 @@
...
 
-```
-pip install rapidata
-```
+They see the following screen:
+
+![Classify Example](./media/order-types/classify-screen.png){ width="40%" }
 
-To update an existing installation:
+## Installation
```
Comment (kannwism) [inline docs/custom_orders.md:12]: Not sure if we need an installation section here

File: docs/custom_orders.md [hunk 4]
```diff
@@ -55,7 +56,7 @@ Set the `workflow` on the `order_builder`:
 order_builder.workflow(workflow)
 ```
 
-3. (Optional) Further configure the order by specifying the number of answers desired for each datapoint (default is 10). Choose either a [`NaiveReferee`](reference/rapidata/rapidata_client/referee/naive_referee.md/#rapidata.rapidata_client.referee.naive_referee.NaiveReferee) or a [`ClassifyEarlyStoppingReferee`](reference/rapidata/rapidata_client/referee/classify_early_stopping_referee.md/#rapidata.rapidata_client.referee.classify_early_stopping_referee.ClassifyEarlyStoppingReferee):
+3. (Optional) Further configure the order by specifying the number of responses<sup>1</sup> desired for each datapoint (default is 10). Choose either a `NaiveReferee` or a `EarlyStoppingReferee`:
```
Comment (kannwism) [inline docs/custom_orders.md:59]: Why deleting links?

File: docs/quickstart_compare.md [hunk 5]
```diff
@@ -42,14 +44,13 @@ order_builder = rapi.create_compare_order("Simple Compare")
 order_builder = order_builder.criteria("Which logo looks better?")
 ```
 
-3. Add the paths to the images you want to classify:
+3. Add the paths to the images you want to classify (order will be randomized):
```
Comment (Makuh17) [inline docs/quickstart_compare.md:47]: I would add an extra comment about how you have to specify the pairs as a list of length 2. I.e., in the end you supply a list of pairs (list of lists). And then the clarification is that the order withing the pair will be randomized when displayed to the user

File: examples/complex_compare_order.py [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
+                      MediaAsset,
+                      PromptMetadata)
 
 def new_compare_order(rapi: RapidataClient):
     logo_path = "examples/data/rapidata_logo.png"
     concept_path = "examples/data/rapidata_concept_logo.jpg"
 
-    # configure validation set
+    # Validation set
+    # This will be shown as defined in the ValidationSelection and will make our annotators understand the task better
```
Comment (Makuh17) [inline examples/complex_compare_order.py:19]: below we set ` question="Which logo is the actual Rapidata logo?",` is this intentional or should it match the other example where we ask which logo is better?

File: scripts/gen_ref_pages_preserve_md_files.py
Comment (Makuh17) [inline scripts/gen_ref_pages_preserve_md_files.py:1]: didnt check this

File: src/rapidata/rapidata_client/order/rapidata_order_builder.py [hunk 7]
```diff
@@ -110,7 +110,7 @@ def _to_model(self) -> CreateOrderModel:
             priority=self._priority,
         )
 
-    def create(self, submit: bool = True, max_workers: int = 10) -> RapidataOrder:
+    def create(self, submit: bool = True, max_workers: int = 10, disable_link=False) -> RapidataOrder:
```
Comment (kannwism) [inline src/rapidata/rapidata_client/order/rapidata_order_builder.py:113]: Please explain new parameter in docstring

**Review by kannwism — CHANGES_REQUESTED** [review]
Looks mostly fine. Why did we remove all the links to generated docs though?

File: docs/media/order-types/classify-screen.png
[the remark this answers is not in the document]
Comment (LinoGiger) [reply docs/media/order-types/classify-screen.png:1]: they are served in the example overview but i do not see any decline in quality

File: docs/quickstart_compare.md [hunk 8]
```diff
@@ -42,14 +44,13 @@ order_builder = rapi.create_compare_order("Simple Compare")
...
 
 ```py
-from rapidata improt MultiAsset, MediaAsset
-order_builder = order_builder.media(MultiAsset([MediaAsset("examples/data/rapidata_logo.png"), MediaAsset("examples/data/rapidata_concept_logo.jpg")]))
+order_builder = order_builder.media([["examples/data/rapidata_logo.png","examples/data/rapidata_concept_logo.jpg"]])
 ```
 
-4. Optionally add additional specifications, here we're adding a specific amount of responses we want per datapoint, there are other functionalities you can explore:
+4. Optionally add additional specifications, here we're adding a specific amount of responses<sup>1</sup> we want per datapoint, there are other functionalities you can explore:
```
[the remark this answers is not in the document]
Comment (LinoGiger) [reply docs/quickstart_compare.md:53]: is this better?
4. Optionally add additional specifications. here we're adding a specific amount of responses<sup>1</sup> we want per datapoint, there are other functionalities you can explore by looking at the order_builder methods:

File: examples/complex_compare_order.py (hunk 6 above)
Comment (LinoGiger) [reply examples/complex_compare_order.py:19]: i see your point but the other question which one is better is subjective. so not a good validation example

File: src/rapidata/rapidata_client/order/rapidata_order.py [hunk 9]
```diff
@@ -71,7 +71,7 @@ def _get_workflow_id(self):
         if self._workflow_id:
             return self._workflow_id
 
-        for _ in range(2):
+        for _ in range(10):
```
[the remark this answers is not in the document]
Comment (LinoGiger) [reply src/rapidata/rapidata_client/order/rapidata_order.py:74]: due to the autovalidation time. there is some issue if you run the order creation and then instantly try to display the progress bar.

File: docs/custom_orders.md (hunk 1 above)
Comment (LinoGiger) [reply docs/custom_orders.md:22]: it didn't work for me anymore

File: docs/custom_orders.md (hunk 2 above)
Comment (LinoGiger) [reply docs/custom_orders.md:2]: yes, but i don't think it hurts? what would you rather have there?

File: docs/custom_orders.md [hunk 10]
```diff
@@ -112,44 +99,37 @@ order = (
...
 
 ```py
 results = order.get_results()
 ```
+
+-------
+
+<sup>1</sup> Due to the possibility of multiple people answering at the same time, this number is treated as a minimum. The actual number of responses may be higher. The overshoot per datapoint will be lower the more datapoints are added.
+
+<sup>2</sup> The Referee is responsible for deciding when to stop collecting responses. The `NaiveReferee` will stop collecting responses after the specified number of responses is reached. The `EarlyStoppingReferee` will stop collecting responses if either the confidence threshold is reached or the max votes have been collected - This is valuable if there is a clear correct answer but less so if it's an oppionion based question.
```
Comment (Makuh17) [inline docs/custom_orders.md:135]:
```suggestion
<sup>2</sup> The Referee is responsible for deciding when to stop collecting responses. The `NaiveReferee` will stop collecting responses after the specified number of responses is reached. The `EarlyStoppingReferee` will stop collecting responses if either the confidence threshold is reached or the max votes have been collected - This is valuable if there is a clear correct answer but less so if it's an opinion based question.
```

File: docs/quickstart_compare.md [hunk 11]
```diff
@@ -42,14 +44,13 @@ order_builder = rapi.create_compare_order("Simple Compare")
...
+3. Add the paths to the images you want to compare. Note that this is a list of lists, where the inner list represents the images that will be compared. The order of those 2 images will be randomized:
 
 ```py
-from rapidata improt MultiAsset, MediaAsset
-order_builder = order_builder.media(MultiAsset([MediaAsset("examples/data/rapidata_logo.png"), MediaAsset("examples/data/rapidata_concept_logo.jpg")]))
+order_builder = order_builder.media([["examples/data/rapidata_logo.png","examples/data/rapidata_concept_logo.jpg"]])
 ```
 
-4. Optionally add additional specifications, here we're adding a specific amount of responses we want per datapoint, there are other functionalities you can explore:
+4. Optionally add additional specifications. here we're adding a specific amount of responses<sup>1</sup> we want per datapoint, there are other functionalities you can explore by looking at the order_builder methods:
```
Comment (Makuh17) [inline docs/quickstart_compare.md:53]:
```suggestion
4. Optionally add additional specifications. Here we're adding a specific amount of responses<sup>1</sup> we want per datapoint, there are other functionalities you can explore by looking at the order_builder methods:
```

**Review by Makuh17 — APPROVED** [review]
A few typos

File: src/rapidata/rapidata_client/order/rapidata_order_builder.py (hunk 7 above)
Comment (kannwism) [reply src/rapidata/rapidata_client/order/rapidata_order_builder.py:113]: .

**Review by kannwism — CHANGES_REQUESTED** [review]
